ksqldb-engine/src/main/java/io/confluent/ksql/physical/scalablepush/consumer/ScalablePushConsumer.java

```diff
@@ -209,6 +262,7 @@ public void closeAsync() {
     for (final ProcessingQueue processingQueue : processingQueues.values()) {
       processingQueue.close();
     }
+    consumer.wakeup();
```

Comment:
is this a bug fix?

Reply:
I moved this to an other PR so shouldn't be a diff here.